### ISSUE TITLE
[question] configuration to only reach out to remote carbon cache instances?

### ISSUE THREAD

**Issue by blysik** [brutasse/graphite-api#212]
Can graphite-api be configured to ONLY check remote carbon cache instances?  I have an architecture where I'd like to put graphite-api on two boxes, which only then reach out to 6 carbon cache instances on other machines.  

Reading the documentation, it seems like you have to have a whisper finder setup?

Thanks

**Comment by blysik** [comment]
I think my question is basically the same as https://github.com/brutasse/graphite-api/issues/207 Can anyone answer this?

**Comment by pkittenis** [comment]
Yes, currently a local whisper setup is included by default if one is not provided, which does not allow for only remote carbon hosts to be configured.

Have made a PR that resolves this by allowing empty local finders configuration, not yet merged - #214

**Comment by sandstheman** [comment]
hi @pkittenis i'm trying to setup a single instance of graphite-api and have it query a pair of carbon-cache instances on a remote server, using your modified code, i can set the finder config to be empty and set the carbon hosts to be the two caches on the remote server, but don't seem to able to get any data being served back.  I'm just getting empty 200 response from graphite-api

**Comment by pkittenis** [comment]
Probably a question for the author, I haven't been using carbon caches for several years now so cannot say without going through the carbon related code in detail. All I can say is that all carbon tests are passing with the modified code.

As far as I can see from a cursory glance, carbon link code is separate from local whisper finder so the two do not seem to be able to affect each other.

Try turning on debug logging, should see messages like `carbonlink request` and `carbonlink finished receiving` if the carbon link connections are sending/receiving data respectively.
